app/middleware/multipleApplications.js

```diff
@@ -132,50 +145,39 @@ const getCase = (req, res) => {
 };
 
 const getDeclarationStatuses = (req, res, next) => {
-    const session = req.session;
-    const formdata = session.form;
+    const formdata = req.session.form;
     const executorsWrapper = new ExecutorsWrapper(formdata.executors);
     const hasMultipleApplicants = executorsWrapper.hasMultipleApplicants();
 
     if ((get(formdata, 'declaration.declarationCheckbox', false)).toString() === 'true' && hasMultipleApplicants) {
-        const ccdCaseId = formdata.ccdCase.id;
+        getCase(req, res, next, true);
+    } else {
+        next();
+    }
+};
 
-        const formData = ServiceMapper.map(
-            'FormData',
-            [config.services.orchestrator.url, req.sessionID]
-        );
+const populateDeclarationFlags = (result, formdata) => {
+    formdata.executorsDeclarations = [];
 
-        formData.getDeclarationStatuses(req.authToken, req.session.serviceAuthorization, ccdCaseId)
-            .then(result => {
-                session.form.executorsDeclarations = [];
-
-                if (result.invitations && result.invitations.length) {
-                    session.form.executorsDeclarations = result.invitations.map(executor => {
-                        let agreed;
-
-                        if (executor.agreed === null) {
-                            agreed = 'notDeclared';
-                        } else if (executor.agreed) {
-                            agreed = 'agreed';
-                        } else {
-                            agreed = 'disagreed';
-                        }
-
-                        return {
-                            executorName: executor.executorName,
-                            agreed: agreed
-                        };
-                    });
+    result.executors.list
+        .forEach((executor, index) => {
+            if (!executor.isApplicant) {
+                const executorAgreed = get(executor, 'executorAgreed');
```

Comment:
can you default the get() method to `null` and leave the check below the same instead of using `typeof` and comparing it to `undefined`?